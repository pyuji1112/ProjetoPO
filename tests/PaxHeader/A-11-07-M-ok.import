29 mtime=1634854036.65464751
