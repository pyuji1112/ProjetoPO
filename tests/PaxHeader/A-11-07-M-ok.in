30 mtime=1634889330.831858405
