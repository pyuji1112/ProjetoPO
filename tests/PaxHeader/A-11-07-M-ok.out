30 mtime=1634889330.342525766
